src/app/components/modal/modal.component.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  CUSTOM_ELEMENTS_SCHEMA,
+  Component, Input,
+} from '@angular/core'
+@Component({
+  selector: 'app-modal',
+  template: `
+    <tds-modal
+      selector="#my-modal-button"
```

Comment:
Since we have the selector attribute here we should now need to manually open the modal with showModal() in the batch table component.

Reply:
i can reproduce the issue with the checkboxes not being checked with the "check all" (sometimes) even if i remove the modal completely.

Reply:
strange, do you mean something like this:

![image](https://github.com/scania-digital-design-system/tegel-angular-demo/assets/84334966/b7bbdea7-b411-4f96-972d-8544a07b93db)

I've tested multiple times (without id but with page refresh & empty cache hard), and sometimes the "check all" works several times in a row, and then it doesn't for several tests

Reply:
i will push a new commit with an id on the table

Reply:
setting an id on the table does not solve the checkbox issue

Reply:
Great, we'll add it as a bug in our backlog

Reply:
Is it possible to release a version of tegel with https://stenciljs.com/docs/angular to ease the integration and maybe invite the teams that uses angular to move over to tegel more quickly?

Reply:
Yes we have it in our plan, so is a React version. But we need to make sure we have a stable vanilla version first.